README.rst

```diff
@@ -33,6 +33,12 @@ You can also see what tzupdate would do without actually doing it by passing
 ``-p``, and specify an alternative IP address by using ``-a``. This is not an
 exhaustive list of options, see ``tzupdate --help`` for that.
 
+The ``contrib/09-timezone`` script can be place in the
```

Comment:
Let's leave this out of the README until there are known users, although it's fine to add it into a `contrib/README.rst`.